### ISSUE TITLE
Nunjucks should have priority over HTML

### ISSUE THREAD

**Issue by code-tree** [alohaas/language-nunjucks#21]
![image](https://cloud.githubusercontent.com/assets/11289719/19192942/408a9210-8cd2-11e6-8b50-cfcadb1b88da.png)

Syntax highlighting doesn't work when brackets are inside an opening tag or its attributes.

(nunjucks syntax is highlighted as red)

**Comment by alohaas** [comment]
Hey @code-tree thanks for the suggestion. I need to ask a clarifying question before making any conclusions or changes. Here's what I'm interpreting from your comment:

You would prefer that nunjucks inserted as html attributes be visually different from other, non-nunjucks html attributes. In other words, using your example, you want`'description'` and `{{ desc }}` rendering in different colors. One final question – are you imagining that `{{ title }}` and `{{ desc }}` should render in the same color?

**Comment by code-tree** [comment]
Yes, that's correct to both. As currently, it seems as though the value in `content="{{ desc }}"` is simply a static string, when in fact it will be replaced by nunjucks.

I assume this is just a matter of giving nunjucks grammar priority over the html grammar, but I wasn't sure how to do that.

**Comment by alohaas** [comment]
Looking into this today.

**Comment by revelt** [comment]
This is a biggie.. Currently good part of syntax highlighting does not happen because of this. For example, this is very apparent when coding inline HTML:

![inline-nunjucks](https://user-images.githubusercontent.com/8344688/31542799-609d7d4e-b00b-11e7-9cc9-03de6d523243.png)

I came here investigating what could be done enable proper syntax highlighting.
